tracer/src/Datadog.Trace/DatabaseMonitoring/VendoredSqlHelpers.cs

```diff
@@ -0,0 +1,103 @@
+// <copyright file="VendoredSqlHelpers.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+
+namespace Datadog.Trace.DatabaseMonitoring
+{
+    internal static class VendoredSqlHelpers
```

Comment:
I wonder if we should try to vendor any _unit tests_ for these methods (and for MultiPartIdentifier)?